apps/docs/src/basics/typography.md

```diff
@@ -0,0 +1,350 @@
+# Typography
+
+::: danger
+Still content to create!
+:::
+
+The onyx font system includes three different font families that can be used throughout your application, pending on the usecase. It contains several headline styles as well as paragraph styles and mono space styles.
+
+::: info
+The font families that can be used with onyx are:
+
+- [Source Sans 3](https://fonts.google.com/specimen/Source+Sans+3)
+- [Source Code Pro](https://fonts.google.com/specimen/Source+Code+Pro?query=source)
+- [Lidl Font Pro](https://www.brand.lidl/brandelements/brandelements/view/id/664#headline_2486)
+- [Kaufland Regular](https://google.com)
+  :::
+
+## Brand Typography
+
+The standard font family is defined to be the Source Sans. If you are creating a new onyx project, this will always be the startingpoint. In the initial phase the Source Sans applies to all color themes that are provided by onyx.
+
+In case you are working on a Kaufland application or Lidl application, you are free to switch into the brand font families of Kaufland Regular or Lidl Font Pro.
+
+::: warning Attention
+Please note that only the `Kaufland Regular` may be implemented in the Kaufland theme and only the `Lidl Font Pro` in the Lidl theme!
+
+For all the other color themes, `Source Sans` remains the font of choice!
+:::
+
+### How to import
+
+To import Kaufland Regular and Lidl Font Pro follow the steps below.
+
+::: code-group
+
+```txt [Kaufland Regular]
+Lorem Ipsum
+```
+
+```txt [Lidl Font Pro]
+Lorem Ipsum
+```
+
+:::
+
+## Font stacks
+
+_custom component for displaying font definitions_
+
+### Headlines
```

Comment:
Can we replace all those details / accordions when we have the custom typography component?

Reply:
Yes this is the intention.